### ISSUE TITLE
[FIXED] error: XMLParser.tag_start: comparison of Fixnum with nil failed

### ISSUE THREAD

**Issue by igroglaz** [rpherbig/dr-scripts#3088]
please help: when I play with lich I got this message every few seconds  '--- error: XMLParser.tag_start: comparison of Fixnum with nil failed' ... It doesn't affect gameplay, but it's pretty annoying.. At least - how to suppress error messages in lich?

log:
https://pastebin.com/6ysiH2PJ

it's the same issue again and again (from logs):
2018-08-25 22:09:43: error: XMLParser.tag_start: comparison of Fixnum with nil failed
    C:/Users/nick/Documents/lich/lich.rbw:1512:in >'
    C:/Users/nick/Documents/lich/lich.rbw:1512:intag_start'
    C:/Ruby200/lib/ruby/2.0.0/rexml/parsers/streamparser.rb:24:in parse'
    C:/Ruby200/lib/ruby/2.0.0/rexml/document.rb:243:inparse_stream'
    C:/Users/nick/Documents/lich/lich.rbw:7201:in `block in open

it happens again and again every few seconds

I've tried to reinstall lich/ruby; tried to run script as admin via cmd - nothing helps

**Comment by igroglaz** [comment]
temporary fixed it with ignoring sentence (thanks to Binu for the idea)

but error still there...

**Comment by igroglaz** [comment]
I fixed this error at my new fresh ruby/lich installation by running Windows command promt (console) with admin rights in Lich folder:
ruby lich.rbw --dragonrealms --stormfront

Hope it would help guys who would meet this issue!

**Comment by rpherbig** [comment]
> Lich Version: 4.6.47

You're using the latest version of Lich, _but not_ the fork with many fixes in it. The latest version of the fork is '4.11.0f'.

In-game, run `;e use_lich_fork`

**Comment by igroglaz** [comment]
sorry I posted my message to the wrong section. Lets continue discussion there:
https://github.com/rpherbig/dr-scripts/issues/3092
p.s.
I did this command on installation, as others:
;repository download dependency.lic
;trust dependency
;dependency install
;e use_lich_fork
;repos download narost-glass.lic
;setupaliases

But yep, it's strange. I did it again, version changed to 4.11.0f' , but char still retreats
